features/font-feature-settings.yml

```
@@ -0,0 +1,10 @@
+name: Font feature settings
+description: The `font-feature-settings` CSS property sets the OpenType feature tags on a font for advanced use cases. When possible, use available `font-variant` options instead.
+spec: https://drafts.csswg.org/css-fonts-4/#font-feature-settings-prop
+group: font-features
+caniuse: font-feature
+compat_features:
+  - css.properties.font-feature-settings
+  - css.properties.font-feature-settings.normal
+  # Not supported in Chrome: https://crbug.com/40398871
+  # - css.at-rules.font-face.font-feature-settings
```

The usual way to handle this with a `compute_from`. Should Chrome resolve that bug, the dist file will automatically change (via Dependaot).

```suggestion
status:
  compute_from: css.properties.font-feature-settings
compat_features:
  - css.properties.font-feature-settings
  - css.properties.font-feature-settings.normal
  - css.at-rules.font-face.font-feature-settings
```